coffee-shop/src/pages/Order/Order.jsx

```diff
@@ -1,11 +1,15 @@
 import React from "react";
 import CustomizedRadioButton from "./components/CustomizedRadioButton/CustomizedRadioButton";
+import GreyHorizontalLine from "../../genericComponents/GreyHorizontalLine";
+import LightOrangeLine from "./components/LightOrangeLine/LightOrangeLine";
```

Comment:
if you added index in this folder like other components and pages, you will be able to import. it without repeating component name 
`import LightOrangeLine from "./components/LightOrangeLine"`